src/simple/simple.go

```diff
@@ -0,0 +1,35 @@
+package simpletest
+
+var b int
+var n int
+
+//Simple вызов
```

Comment:
```suggestion
// Simple вычисляет все простые числа в диапазоне от 0 до N
```

так более по-гошному :)